kafka-streams/src/main/java/MathMethods.java

```diff
@@ -0,0 +1,29 @@
+import java.util.List;
+
+public class MathMethods {
+    public static double sum(List<Double> a){
```

Comment:
where is this being used?

Reply:
I mean method sum

Reply:
method sum is being used inside method 'mean'